index.html

```diff
@@ -5,11 +5,86 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
-    <title>Document</title>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet"/>
+    <title>My portfolio</title>
 </head>
 <body>
     
-    <h1>Hola Mundo</h1>
+    <header>
+        <nav class="header" id="header">
+          <ul>
+            <li class="nav-item">
+              <a href="#projects">Portfolio</a>
+            </li>
+            <li class="nav-item">
+              <a href="#about">About</a>
+            </li>
+            <li class="nav-item">
+              <a href="#contact">Contact</a>
+            </li>
+          </ul>
+  
+          <div class="burger" class="burger">
+            <span class="bar top"></span>
+            <span class="bar mid"></span>
+            <span class="bar bot"></span>
+          </div>
+        </nav>
+      </header>
+      <section class="main" id="main">
+        <div class="main-container">
+          <h1>
+            Hey there. <span class="span-br"><br></span>
+            I’m Alex.
+          </h1>
+          <span class="h1-sub">I’m a software developer</span>
```

Comment:
- [ ] The subtitle (if existing) uses a header tag hx (h2, ... h6) except `h1` which has been used for the main title. This is semantically correct and the screen readers and other technologies will thank you. Kindly change the subtitle to any of the above mentioned hx tags.